src/Database/Type/IntegerType.php

```diff
@@ -80,10 +81,33 @@ public function toDatabase($value, Driver $driver)
     public function toPHP($value, Driver $driver)
     {
         if ($value === null) {
-            return null;
+            return $value;
         }
 
-        return (int)$value;
+        // Using coercion is faster than casting
+        // @codingStandardsIgnoreStart
+        return (int)+$value;
```

Comment:
Still remains. Though it is `(int)+$value`, it is slower than `(int)$value` in PHP 7.x at least.

Reply:
This would need to be `(int) + $value` then for coding standards?
But is this new syntax really necessary given the fact that this is quite the nano opt. somewhat? Only will really a looot of casts this might affect in the micro second area right?